andela/suyabay
# #122028319 users should be redirected to their account after signup

#### What does this PR do?
- Redirected users to their account after registration.
- Replace the misleading notification with the relevant notification.
#### Description of Task to be completed?
- Modify the `LoginAndSignUp.js` file to edit the swal message from `send confirmation email` to `Check your email for relevant information`.
- Modify the `postRegister` method in the `AuthController.php` to log in the user after a successful registration.
#### How should this be manually tested?
- Go to `http://suyabay-staging.herokuapp.com/` then click on `Sign Up` link. This will take you to the signup page. Fill the required information and click on the `Sign Up` button, a dialog box showing the notification: `{username} Your SuyaBay account has been successfully created Check your email for relevant information`. Click on the`ok` button and you will be redirected to the homepage already logged in.
#### Any background context you want to provide?

none
#### What are the relevant pivotal tracker stories?
#122028319
#### Screenshots (if appropriate)

![screen shot 2016-06-22 at 1 27 50 pm](https://cloud.githubusercontent.com/assets/14159420/16266612/a271cb3c-387d-11e6-8f4e-a173a32b3095.png)
#### Questions:

none

[@unicodeveloper](https://github.com/unicodeveloper)
[@andela-oadebayo](https://github.com/andela-oadebayo)
